src/main/java/com/minecolonies/core/items/ItemScrollColonyTP.java

```diff
@@ -133,6 +134,10 @@ public void appendHoverText(
         {
             colonyDesc = Component.literal(colony.getName());
         }
+        else if (stack.getOrCreateTag().contains(TAG_CACHED_COLONY_NAME))
+        {
+            colonyDesc = Component.literal(stack.getOrCreateTag().getString(TAG_CACHED_COLONY_NAME));
```

Comment:
we actuall yhad that before and removed it since it doesnt adjust to changed names xD

Reply:
nvm